bitcoin/src/psbt/error.rs

```diff
@@ -180,7 +185,7 @@ impl std::error::Error for Error {
             | Taproot(_)
             | XPubKey(_)
             | Version(_)
-            | PartialDataConsumption=> None
+            | PartialDataConsumption=> None,
```

Comment:
If you have to rebase for some other reason:

```suggestion
            | PartialDataConsumption => None,
```

Reply:
But actually, from a purest perspective this change should not even be included.

Reply:
I saw this dangling after Andrew's Ack 🙉

Reply:
Not to worry :)